README.rst

```diff
@@ -74,8 +74,7 @@ https://docs.scrapy.org/en/latest/intro/install.html for more details.
 Documentation
 =============
 
-Documentation is available online at https://docs.scrapy.org/ and in the ``docs``
-directory.
+Documentation is available online at https://docs.scrapy.org/ and in the `docs <https://github.com/scrapy/scrapy/blob/master/docs>`_ directory.
```

Comment:
For me this URL redirects to https://github.com/scrapy/scrapy/tree/master/docs - would it make sense to use the final url, and avoid the redirect?

That said, I'm not sure we should be pointing users to `docs` folder; it doesn't render properly on github, because github doesn't have full sphinx support. What do you think about remoing this note instead, and pointing only to https://docs.scrapy.org/? cc @Gallaecio.

Reply:
I can revert this line and open a new PR where we can continue the discussion.

Reply:
Opene https://github.com/scrapy/scrapy/pull/5663 for further discussion